core/domain/user_services.py

```diff
@@ -656,3 +656,38 @@ def get_user_dashboard_stats(user_id):
             'total_plays': 0,
             'average_ratings': None
         }
+
+def get_weekly_dashboard_stats(user_id):
```

Comment:
(sorry for the design flip-flop)

Thinking even more, I think it's actually good to store everything -- it might be interesting to plot some graphs for the user to see their creator history, at some point in time.

In that case, could you update the docstring to mention that the length of the list is the number of weeks since the user has created an exploration with plays/ratings?